app/scripts/views/settings/client_disconnect.js

```diff
@@ -13,7 +13,6 @@ define(function (require, exports, module) {
   const t = require('views/base').t;
   const Template = require('stache!templates/settings/client_disconnect');
 
-  const REASON_SELECTOR = '.disconnect-reasons';
```

Comment:
@vladikoff : It was only used https://github.com/mozilla/fxa-content-server/pull/4578/files#diff-1c0ead66105fbf6c8ef3c4ed4053c330R81. I directly used '.disconnect-reasons' itself.